frontend/src/modules/editor/components/Editor.css

```diff
@@ -37,26 +32,25 @@
     font-size: 15px;
 }
 
-.editor .banner {
+.banner {
     font-style: italic;
     line-height: 40px;
 }
 
-.editor .banner a {
+.banner a {
     color: #7BC876;
 }
 
-.editor menu {
+.editor-menu {
```

Comment:
I'd rather use `.editor > menu` if you want to enforce a stricter rule. I think we should not abuse CSS classes when they're not needed.